client/src/rpc_response.rs

```diff
@@ -511,7 +511,7 @@ pub enum RpcBlockUpdateError {
     UnsupportedTransactionVersion(u8),
 }
 
-#[derive(Serialize, Deserialize, Debug)]
+#[derive(Serialize, Deserialize, Debug, PartialEq)]
```

Comment:
Why no `Eq` or `Clone` here?

Reply:
I'll add Clone (missed it), can't derive Eq because there are floats in the underlying structs if you go deep enough